spec/integration/transport/winrm_spec.rb

```diff
@@ -211,10 +203,11 @@ def stub_winrm_to_raise(klass, message)
     end
   end
 
-  context "authenticating with Kerberos", kerberos: true do
+  context "authenticating with Kerberos", kerberos: true, omi: true do
     before(:all) do
       # disable all tests on Windows for now
       skip('Windows Active Directory tickets are different') if Bolt::Util.windows?
+      skip("WinRM gem and OMI server have a protocol negotiation bug")
```

Comment:
One of the 3 tests was working previously... but now they're all skipped. Not sure I understand this change.

Reply:
The only test that ran before just ensured that running `kinit` locally had worked as a precondition for the subsequent tests, which were marked pending. So I just skipped the whole section instead.